micro-rdk/src/common/provisioning/server.rs

```diff
@@ -35,6 +35,27 @@ use hyper::{
 };
 use prost::Message;
 
+#[derive(Default, Clone)]
```

Comment:
Did this need to move up here? If not and it is just fallout, I'd recommend moving it back to its original location.

Reply:
moved back